### PR TITLE
Added Solutions folder

### PR DESCRIPTION
**Pull request by gvarao**
Added use cases rules and removed while spaces from system-process.py file

### CODE REVIEW

File: juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: 402
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml
  8:1       error    trailing spaces  (trailing-spaces)
  12:1      error    trailing spaces  (trailing-spaces)
  17:1      error    trailing spaces  (trailing-spaces)

File: juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: 402
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml
  5:9       error    wrong indentation: expected 4 but found 8  (indentation)
  5:20      warning  truthy value is not quoted  (truthy)
  8:9       error    wrong indentation: expected 4 but found 8  (indentation)
  12:1      error    trailing spaces  (trailing-spaces)
  15:16     error    too many spaces after colon  (colons)
  16:16     error    too many spaces after colon  (colons)
  17:16     error    too many spaces after colon  (colons)

Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: 403
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml
  5:16      warning  truthy value is not quoted  (truthy)
  12:1      error    trailing spaces  (trailing-spaces)

File: juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml (hunk 1 above)
Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: 405
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml
  8:1       error    trailing spaces  (trailing-spaces)
  12:1      error    trailing spaces  (trailing-spaces)
  17:1      error    trailing spaces  (trailing-spaces)

File: juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml (hunk 2 above)
Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: 405
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml
  5:16      warning  truthy value is not quoted  (truthy)
  12:1      error    trailing spaces  (trailing-spaces)

File: juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml (hunk 1 above)
Comment (ayushgithub) [inline juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: 406
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml
  8:1       error    trailing spaces  (trailing-spaces)
  12:1      error    trailing spaces  (trailing-spaces)
  17:1      error    trailing spaces  (trailing-spaces)

Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: Addressed and resolved

File: juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml (hunk 2 above)
Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: Addressed and resolved

Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: Addressed and resolved

File: juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
```
[the remark this answers is not in the document]
Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: Addressed and resolved

File: juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml (hunk 1 above)
Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: Addressed and resolved

File: juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml (hunk 2 above)
Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/rsvpsession.yml]: Addressed and resolved

File: juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml (hunk 1 above)
Comment (gvarao) [reply juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml]: Addressed and resolved

**Pull request merged**